scripts/password_filter.py

```diff
@@ -1,16 +1,59 @@
 import re
-def password_filter(src, dest, regex):
-	pattern = re.compile(regex)
-	pw_list = []
+import sys
+
+all_digits_regex = re.compile('^\d+$')
+def is_all_digits(password):
+    return all_digits_regex.match(password)
+
+all_alpha_regex = re.compile('^[a-z]+$')
+def is_all_alpha(password):
+    return all_alpha_regex.match(password)
+
+def password_filter(src, dest):
+        pw_set = set([])
+
 	with open(src, 'r') as f:
 		for line in f:
-			m = re.match(pattern, line)
-			if(m):
-				print m.group(1)
-				pw_list.append(m.group(1))
-	print len(pw_list)
+                # Several checks are done before the bloom filter is called:
```

Comment:
Instead of a single regexp, follow the rules we use in passwordcheck.js to remove possibilities that have already been flagged.